# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../chrony/spinlock_h_remove.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- chrony-1.20/rtc_linux.c~	2003-09-22 23:22:30.000000000 +0200
+++ chrony-1.20/rtc_linux.c	2004-10-15 13:10:45.340778240 +0200
@@ -43,12 +43,7 @@
 #include <sys/time.h>
 #include <sys/types.h>
 
-#ifdef HAS_SPINLOCK_H
-#include <linux/spinlock.h>
-#else
-/* Include dummy definition of spinlock_t to cope with earlier kernels. */
-typedef int spinlock_t;
-#endif
+/* not user space business to include linux/spinlock.h , so removed */
 
 /* This is a complete hack since the alpha sys/io.h needs these types
  * but does not arrange them to be defined.  This is almost certainly
